### PR TITLE
[ADD] helpdesk_delivery

### PR DESCRIPTION
**Pull request by osi-scampbell**
This is PR adds the helpdesk_delivery module to the osi-addons repo. This module mimics the fieldservice_delivery module.

### CODE REVIEW

File: helpdesk_delivery/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2018 - TODAY, Brian McMaster
```
Comment (max3903) [inline helpdesk_delivery/__init__.py]:
```suggestion
# Copyright (C) 2020 Open Source Integrators
```

File: helpdesk_delivery/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2018 - TODAY, Brian McMaster
```
Comment (max3903) [inline helpdesk_delivery/__manifest__.py]:
```suggestion
# Copyright (C) 2020 Open Source Integrators
```

File: helpdesk_delivery/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Helpdesk - Delivery',
+    'summary': 'Select delivery methods and carriers on Helpdesk Tickets',
+    'version': '12.0.1.0.0',
+    'category': 'Helpdesk',
+    'author': "Open Source Integrators, "
+              "Odoo Community Association (OCA)",
```
Comment (max3903) [inline helpdesk_delivery/__manifest__.py]:
```suggestion
    'author': "Open Source Integrators",
```

File: helpdesk_delivery/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Copyright (C) 2018 - TODAY, Brian McMaster
```
Comment (max3903) [inline helpdesk_delivery/models/__init__.py]:
```suggestion
# Copyright (C) 2020 Open Source Integrators
```

File: helpdesk_delivery/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (max3903) [inline helpdesk_delivery/__init__.py]:
```suggestion
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

File: helpdesk_delivery/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2018 - TODAY, Brian McMaster
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (max3903) [inline helpdesk_delivery/__manifest__.py]:
```suggestion
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

File: helpdesk_delivery/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    'depends': [
+        'helpdesk_stock',
+        'delivery',
+    ],
+    'data': [
+        'views/helpdesk_ticket.xml',
+    ],
+    'installable': True,
+    'auto_install': True,
+    'license': 'AGPL-3',
```
Comment (max3903) [inline helpdesk_delivery/__manifest__.py]:
```suggestion
    'license': 'LGPL-3',
```

File: helpdesk_delivery/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Helpdesk - Delivery',
+    'summary': 'Select delivery methods and carriers on Helpdesk Tickets',
+    'version': '12.0.1.0.0',
+    'category': 'Helpdesk',
+    'author': "Open Source Integrators, "
+              "Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/field-service',
```
Comment (max3903) [inline helpdesk_delivery/__manifest__.py]:
```suggestion
    'website': 'https://github.com/ursais/osi-addons',
```

File: helpdesk_delivery/models/__init__.py [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (max3903) [inline helpdesk_delivery/models/__init__.py]:
```suggestion
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

File: helpdesk_delivery/models/stock_request.py [hunk 10]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2020, Open Source Integrators
```
Comment (max3903) [inline helpdesk_delivery/models/stock_request.py]:
```suggestion
# Copyright (C) 2020 Open Source Integrators
```

File: helpdesk_delivery/views/helpdesk_ticket.xml [hunk 11]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline helpdesk_delivery/views/helpdesk_ticket.xml]:
```suggestion
```

**Comment by max3903** [comment]
@osi-scampbell Can you add this module to the exclude list of Travis?

**Review by max3903 — APPROVED** [review]

**Comment by osi-scampbell** [comment]
@max3903 Changes added and squashed

**Pull request merged**